README.md

```diff
@@ -0,0 +1,4 @@
+Description
+===========
+
+The ip_isr package provides Instrument Signature Removal related tasks. ISR includes steps such as combining multiple amplifiers into one full CCD image,  corrections for overscans, crosstalk, bias and dark frames, and the creation of variance and mask planes.
```

Comment:
Need to update the copy-pasta!

(...and I'd rather like this before I get too far into this review so that I know what the package is advertising!)